taca_ngi_pipeline/deliver/deliver.py

```diff
@@ -87,6 +87,12 @@ def __init__(self, projectid, sampleid, **kwargs):
         self.reportpath = getattr(self, 'reportpath', None)
         self.force = getattr(self, 'force', False)
         self.stage_only = getattr(self, 'stage_only', False)
+        #Fetches a project name, should always be availble; but is not a requirement
+        try:
+            n = db.project_entry(db.dbcon(), projectid)['name']
+            self.projectname = n
+        except KeyError:
+            pass
```

Comment:
I know I wrote the code below so I am the one to blame.. but for the record, I don't like that the database is consulted in the constructor